### PR TITLE
(Update) PHP8 Support

### PR DESCRIPTION
**Pull request by HDVinnie**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes

### CODE REVIEW

**Comment by Ne-Lexa** [comment]
Thanks for your contribution.
Add PHP 8.0 support testing to Travis.

**Comment by HDVinnie** [comment]
> Thanks for your contribution.
> Add PHP 8.0 support testing to Travis.

Done

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
         }
     ],
     "require": {
-        "php": "^5.5.9 || ^7.0",
+        "php": "^7.2 | ^7.3 | ^7.4 | ^8.0",
```
Comment (Ne-Lexa) [inline composer.json:24]: No need to remove php 5.5 support.

```json
        "php": "^5.5.9 | ^7.0 | ^8.0",
```

File: .travis.yml [hunk 2]
```diff
@@ -41,6 +41,11 @@ matrix:
       os: linux
       dist: xenial
       env: COVERAGE=true ZIPALIGN_INSTALL=true PHPUNIT_FLAGS="-v -c phpunit.xml --coverage-clover=coverage.clover"
+      
+    - php: 8.0
```
Comment (Ne-Lexa) [inline .travis.yml:45]: While travis hasn't added php8 yet, you can use "php8.0snapshot"

File: .travis.yml [hunk 3]
```diff
@@ -41,6 +41,11 @@ matrix:
...
+      os: linux
+      dist: xenial
+      env: COVERAGE=true ZIPALIGN_INSTALL=true PHPUNIT_FLAGS="-v -c phpunit.xml --coverage-clover=coverage.clover"
```
Comment (Ne-Lexa) [inline .travis.yml:48]: For only one version, code coverage should be considered. I usually leave the cover on the latter.
Remove coverage from php7.4.

**Comment by HDVinnie** [comment]
@Ne-Lexa anything else need to be done?

**Comment by Ne-Lexa** [comment]
The Travis check fails. We need to adapt tests for PHP 8.

**Comment by HDVinnie** [comment]
Can you link your travis to this repo so I can see where it's failing? Or provide a link? Ill happily review and PR.

**Comment by giansalex** [comment]
@Ne-Lexa can you apply this change? 
at least for allow `composer install`

**Comment by Ne-Lexa** [comment]
Version 4.0 has been released with PHP 8 support.
Thank you for your contribution.

**Pull request closed without merging**